testsuite/features/step_definitions/command_steps.rb

```diff
@@ -9,6 +9,11 @@
   node.run("rpm -q #{package}")
 end
 
+Then(/^Deb package "([^"]*)" with version "([^"]*)" should be installed on "([^"]*)"$/) do |package, version, host|
```

Comment:
Debian package ? 

"Debian" is the official name, so if we could agree on not using DEB, Deb, whatever...

Reply:
> In this case it "DEB" refers to the pkg format, same as "RPM" in the previous scenario.

So the distinction between "deb" and "Debian" should be clear with this remark, but I think in some places in general, "Ubuntu" vs. "Debian" is also a problem.

Reply:
> So the distinction between "deb" and "Debian" should be clear with this remark, but I think in some places in general, "Ubuntu" vs. "Debian" is also a problem

@cbbayburt I changed all Debian references to Ubuntu to be more consistent. Only kept `Deb` in the channel name to indicate the type of the packages.